### ISSUE TITLE
Git Submodule piet-gpu

### ISSUE THREAD

**Issue by jonas-w** [flxzt/rnote#257]
**Describe the bug**
The piet-gpu submodule is missing in the .gitmodules file.

**To Reproduce**
1. Clone the repo
2. execute git submodule update --init --recursive

**Expected behavior**
Inits all submodules

** Console Output **
```
Submodule 'piet' (https://github.com/flxzt/piet) registered for path 'piet'
fatal: No url found for submodule path 'piet-gpu' in .gitmodules
```

**Comment by flxzt** [comment]
Thanks for reporting! Something went wrong with merging yesterday. Should be fixed with b2acdfbed63fb113093f820ebf0e5aaed95f7560 .

**Comment by jonas-w** [comment]
That was fast! Thank you! Seems to be working.
